api/src/main/resources/workbench.yaml

```diff
@@ -417,6 +418,31 @@ paths:
           schema:
             $ref: "#/definitions/EmptyResponse"
 
+  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/share:
+    post:
+      tags:
+        - workspaces
+      description: Shares a workspace with users
+      operationId: shareWorkspace
+      parameters:
+          - in: path
+            name: workspaceNamespace
+            type: string
+            required: true
+          - in: path
+            name: workspaceId
+            type: string
+            required: true
+          - in: body
+            name: users
```

Comment:
I'd prefer users_and_roles or something, since it's not just user entities.

Reply:
Will change.